keel-api/src/main/kotlin/com/netflix/spinnaker/keel/plugin/TaskLauncher.kt

```diff
@@ -30,12 +29,12 @@ interface TaskLauncher {
   suspend fun submitJob(
     user: String,
     application: String,
-    notifications: List<OrcaNotification>,
+    notifications: Collection<NotificationConfig>,
```

Comment:
Shouldn't this be a `Set` instead?

Reply:
Do you mean because we want to guarantee uniqueness of elements?

Reply:
Yes. I would expect we wouldn't allow duplicate/redundant notification configs, in this case.